<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Member</class>
 <widget class="QWidget" name="Member">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>346</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

background:#7898C4;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Mon Name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>257</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-6</number>
    </property>
    <property name="maximum">
     <number>6</number>
    </property>
   </widget>
   <widget class="QLabel" name="spdefLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>208</y>
      <width>60</width>
      <height>39</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SPDEF</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-6</number>
    </property>
    <property name="maximum">
     <number>6</number>
    </property>
   </widget>
   <widget class="QLabel" name="atkLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ATK</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>208</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-6</number>
    </property>
    <property name="maximum">
     <number>6</number>
    </property>
   </widget>
   <widget class="QLabel" name="defLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>60</width>
      <height>39</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DEF</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="spatkLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>159</y>
      <width>60</width>
      <height>39</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SPATK</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-6</number>
    </property>
    <property name="maximum">
     <number>6</number>
    </property>
   </widget>
   <widget class="QLabel" name="spdLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>257</y>
      <width>60</width>
      <height>39</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SPD</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>159</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-6</number>
    </property>
    <property name="maximum">
     <number>6</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="health">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>306</y>
      <width>620</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>10000</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
    <property name="format">
     <string>%p%</string>
    </property>
   </widget>
   <widget class="QLabel" name="hpLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>HP</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="hpInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="sprite">
    <property name="geometry">
     <rect>
      <x>534</x>
      <y>10</y>
      <width>96</width>
      <height>96</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: black
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>116</y>
      <width>480</width>
      <height>180</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-width: 1px;
border-color: #3264a8</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>251</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="Move" name="move1" native="true">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>126</y>
      <width>225</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="Move" name="move3" native="true">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>211</y>
      <width>225</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="Move" name="move2" native="true">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>126</y>
      <width>225</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="Move" name="move4" native="true">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>211</y>
      <width>225</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <zorder>name</zorder>
   <zorder>spinBox_5</zorder>
   <zorder>spdefLabel</zorder>
   <zorder>spinBox</zorder>
   <zorder>atkLabel</zorder>
   <zorder>spinBox_4</zorder>
   <zorder>defLabel</zorder>
   <zorder>spatkLabel</zorder>
   <zorder>spinBox_2</zorder>
   <zorder>spdLabel</zorder>
   <zorder>spinBox_3</zorder>
   <zorder>health</zorder>
   <zorder>hpLabel</zorder>
   <zorder>hpInput</zorder>
   <zorder>sprite</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>move4</zorder>
   <zorder>move2</zorder>
   <zorder>move1</zorder>
   <zorder>move3</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Move</class>
   <extends>QWidget</extends>
   <header location="global">../resources/move.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
